detections/endpoint/suspicious_msbuild_path.yml

```diff
@@ -0,0 +1,43 @@
+name: Suspicious msbuild path
+id: f5198224-551c-11eb-ae93-0242ac130002
+version: 1
+date: '2021-01-12'
+description: The following analytic identifies msbuild.exe executing from a non-standard
+  path. Msbuild.exe is natively found in C:\Windows\Microsoft.NET\Framework\v4.0.30319
+  and C:\Windows\Microsoft.NET\Framework64\v4.0.30319. A moved instance of MSBuild
+  is suspicious, however there are instances of build applications that will move
+  or use a copy of MSBuild.
+how_to_implement: To successfully implement this search you need to be ingesting information
+  on process that include the name of the process responsible for the changes from
+  your endpoints into the `Endpoint` datamodel in the `Processes` node.
+type: ESCU
+references:
+- https://lolbas-project.github.io/lolbas/Binaries/Msbuild/
+- https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1127.001/T1127.001.md
+author: Michael Haag, Splunk
+search: '| tstats `security_content_summariesonly` count values(Processes.process_name)
+  as process_name values(Processes.process) as process min(_time) as firstTime max(_time)
+  as lastTime from datamodel=Endpoint.Processes where Processes.process_name=msbuild.exe
+  AND (Processes.process_path!=c:\\windows\\microsoft.net\\framework*\\v4.0.30319\\*)
```

Comment:
Can we make that independent from the version?

Reply:
Removed the specified version and updated false positive and description to reflect Visual Studio usage of `msbuild.exe`.